components/script/dom/bluetoothremotegattserver.rs

```diff
@@ -46,6 +46,34 @@ impl BluetoothRemoteGATTServer {
     fn get_bluetooth_thread(&self) -> IpcSender<BluetoothRequest> {
         self.global().as_window().bluetooth_thread()
     }
+
+    pub fn set_connected(&self, connected: bool) {
+        self.connected.set(connected);
+    }
+
+    // https://webbluetoothcg.github.io/web-bluetooth/#garbage-collect-the-connection
+    #[allow(unrooted_must_root)]
+    fn garbage_collect_the_connection(&self) -> ErrorResult {
```

Comment:
I put this function here because in the specification [this function](https://webbluetoothcg.github.io/web-bluetooth/#garbage-collect-the-connection) is in the section of the `BluetoothRemoteGATTServer` methods, but it can be moved to BluetoothDevice methods as well.